src/core/doc_generators/models.py

```
@@ -42,45 +44,40 @@ class DocumentGeneratorService(BaseModel):
     このクラスは、ドキュメント生成処理のビジネスロジックを実装します。
     """
 
-    def generate_document(
-        self, config: DocumentConfig, **kwargs: Any
-    ) -> GenerationResult:
+    # DocumentGeneratorProtocolとの互換性を確保（objectを使用）
```

Ruff (0.14.0)

Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)